src/sanity_checker.rs

```diff
@@ -33,7 +34,7 @@ pub fn check_config_option_sanity(config_options: &Input) -> Result<(), String>
         is_input_sane = false;
     }
 
-    if !check_ports_are_valid_with_host(
```

Comment:
Unsure if you're actually expecting `receive_tcp_acars` to be an array of host:port or just ports. The help text indicates that it should just be ports, but the sanity check you were using was trying to do host AND port validation..?

Reply:
The documentation in the `Input` struct is wrong; the input should be `host:port`. That is my mistake. I'd like to blame GitHub Copilot but I suppose I could have done my due diligence checking to make sure it was right :)

The README.md (`acars_router/README.md`) has it right.